udata_hydra/utils/http.py

```diff
@@ -2,12 +2,21 @@
 import logging
 
 import aiohttp
+from aiohttp import web
 
 from udata_hydra import config
 
 log = logging.getLogger("udata-hydra")
 
 
+def get_request_params(request, params_names: list[str]) -> list:
```

Comment:
Good call on making `params_names` explicit, without the default value `_get_args` had, which was too implicit to my taste 👍